code/modules/client/preference_setup/vore/07_traits.dm

```diff
@@ -135,6 +161,11 @@
 			var/datum/trait/trait = negative_traits[T]
 			. += "<li>- <a href='?src=\ref[src];clicked_neg_trait=[T]'>[trait.name] ([trait.cost])</a></li>"
 		. += "</ul>"
+	else if (!(pref.species == SPECIES_XENOCHIMERA))
+		. += "<b>Enable Trash Can:</b> <a [pref.trait_trash_eater ? "class='linkOn'" : ""] href='?src=\ref[src];toggle_trait_trash_eater=0'><b>[pref.trait_trash_eater ? "Yes" : "No"]</b></a><br>"
```

Comment:
This is the kind of thing I don't want to see, for example. I don't want everyone suddenly being able to eat garbage. There's no logical reason for this among most races.

Reply:
xenochimera are obligate carnivores, if there was a reagent tag for it I'd make them unable to gain nutrition from regular nutriment, never mind from literal fucking garbage